_std/machinery.dm

```diff
@@ -18,11 +18,13 @@
 #define TOTAL 4	//for total power used only
 
 // bitflags for machine stat variable
-#define BROKEN 1		// machine non-functional
-#define NOPOWER 2		// no available power
-#define POWEROFF 4		// machine shut down, but may still draw a trace amount
-#define MAINT 8			// under maintainance
-#define HIGHLOAD 16		// using a lot of power
+#define BROKEN    (1<<0)		// machine non-functional
```

Comment:
Can you put these in the DMdoc style

Reply:
Sure! 
I find code diving faster and less error prone so I never spent much looking at the Generated Docs, so I guess I'll try to stop being part of the problem. 😉